### DISCUSSION TITLE
[Question] xf86-input-wacom or opentabletdriver

### DISCUSSION THREAD

**Discussion by Zerogaku** [OpenTabletDriver/OpenTabletDriver#3145] [General]
Hi I have a wacom intuos pro and have been thinking of using one of the 2 options to set built in keybindings on the tablet, I'm just not sure which to pick, i know osu players like to use `opentabletdriver` but i use my tablet to draw.

Currently the only difference for me is that  <ins>`xf86-input-wacom` supports the touch ring</ins> for _2 scroll functions_ (up or down), I know the touch ring could potentially hold 4 macros, but currently `xf86-input-wacom` does not support such functionality and probably not for a while, I personally prefer to use the mouse instead. 

And  <ins>`OpenTabletDriver` does not yet support the touch ring</ins>, but its configuration is **much more** user friendly, also it works on wayland and I might use it as a daily driver in the future.

Does `opentabletdriver` offer any advantages in performance over the default wacom driver that x11 provides?

**Comment by jamesbt365** [comment]
OpenTabletDriver offers lots of customization, but as computers are very fast, there isn't exactly a performance difference.
